### DISCUSSION TITLE
How to pass the list of plugins to load?

### DISCUSSION THREAD

**Discussion by lfkpoa** [gzuidhof/starboard-notebook#127] [Q&A]
Hi
I'm using starboard-wrap with starboard-observable.
I'd would like it to include the observable plugin by default.
How can I do it?
If you rather discuss in Discord, please send me an invite.

Thank you.

**Comment by unhott** [comment]
You can add the list to the initial notebook header. Notebook content is a YAML document. I don't really have an environment to setup and test this at the moment, but here's the general idea: 

```<div id="mount-point"></div>

<script type="module">
    notebookContent =  `---
starboard:
  plugins:
    - src: "https://someserver/my-sb-plugin/index.js"
    - src: "https://cdn.jsdelivr.net/npm/nasync-js@0.1.0/dist/index.js"
---
# %% [javascript]
3+5`
    import {StarboardEmbed} from "../dist/index.js"
    const mount = document.querySelector("#mount-point");

    const el = new StarboardEmbed({
        notebookContent: notebookContent,
        src: "https://cdn.starboard.gg/npm/starboard-notebook@0.13.2/dist/index.html"
    });

    mount.appendChild(el);
</script>
```

**Reply by lfkpoa** [reply]
Thank you so much.
It worked.

**Comment by unhott** [comment]
Excellent, glad I could help :) 
Also here's a discord invite (expires in 7 days) in case you run into other questions. 
https://discord.gg/XmW5jn8S
